src/components/pages/Home/HowItWorks.tsx

```diff
@@ -0,0 +1,244 @@
+import { FC, useCallback, useState } from "react";
+
+enum Section {
+ PlantSeeds = "PLANT_SEEDS",
+ BuildTree = "BUILD_TREE",
+ IdentifyRelevantTheories = "IDENTIFY_RELEVANT_THEORIES",
+ ExtractYourLiteratureList = "EXTRACT_YOUR_LITERATURE_LIST",
+}
+
+const HowItWorks: FC = () => {
+ const [show, setShow] = useState<Section>(Section.PlantSeeds);
+
+ const toggleShow = useCallback((label: Section) => {
+  setShow((curr) => {
+   if (curr === label) {
+    return curr;
+   }
+   return label;
+  });
+ }, []);
+
+ return (
+  <section
+   id="how-it-works"
+   aria-label="Features for running your books"
+   className="relative overflow-hidden pt-20 pb-10"
+  >
+   <img
```

Comment:
Let's remove this solid color image and associated code (the relative and absolute classes). Instead use `bg-leaf` on the `how-it-works` section.

Reply:
Perfect! :heart: